stratum/hal/bin/barefoot/BUILD

```diff
@@ -47,7 +47,7 @@ stratum_cc_binary(
 pkg_tar(
     name = "stratum_binaries",
     srcs = [
-        "deb/stratum-entrypoint",
+        "deb/stratum-entrypoint.sh",
```

Comment:
I think the extension is unnecessary, but for consistency either all or none should have them. For now it's all.